README.md

```diff
@@ -161,8 +169,12 @@ Once you are able to connect **LambdaTest Tunnel** successfully, you would just
 **Tunnel Capability**
 
 ```java
-DesiredCapabilities capabilities = new DesiredCapabilities();        
-        capabilities.setCapability("tunnel", true);
+ChromeOptions browserOptions = new ChromeOptions();         
+        browserOptions.setCapability("platformName", "Windows 10");
```

Comment:
@haiderLambdatest This code snippet is not according to Selenium4 capabilities. Please review and check.